boards/shields/ssd1306/sh1106_128x64.overlay

```diff
@@ -25,5 +25,6 @@
 		segment-remap;
 		com-invdir;
 		prechargep = <0x22>;
+		sh1106-compatible;
```

Comment:
Please remove `sh1106-compatible;` property.

```
-- Found devicetree overlay: /home/jofi/zephyr-project/zephyr/boards/shields/ssd1306/sh1106_128x64.overlay
devicetree error: 'sh1106-compatible' appears in /soc/i2c@40066000/ssd1306@3c in /home/jofi/zephyr-project/zephyr/samples/subsys/display/cfb/outdir/zephyr/zephyr.dts.pre, but is not declared in 'properties:' in /home/jofi/zephyr-project/zephyr/dts/bindings/display/sinowealth,sh1106-i2c.yaml
```

Reply:
Thank you for pointing out.
It's my careless mistake.
I confirmed that the compilation passed with the following command.

```
 west build -b frdm_k64f -d ../k64f samples/subsys/display/lvgl/ -- -DSHIELD=sh1106_128x64
```